### ISSUE TITLE
Improve regular fire detection

### ISSUE THREAD

**Issue by black-roland** [chuck-norris-network/teeworlds#7]
Do not ban players just for rapid-fire if they do not use other cheats.

**Comment by derselbst** [comment]
Did that actually happen?

**Comment by black-roland** [comment]
> Did that actually happen?

Yeah :( I'll disable this check for some time to avoid false positives. Well, perhaps we should remove this check or ignore when warning level == 0 or something better.

**Comment by derselbst** [comment]
Ok, then disable it for now. I'll look over it and see whether it can be made more reliable.
